Jenkinsfile

```diff
@@ -1,3 +1,7 @@
+def SKIP_SIGNING_DEFAULT = true
+if (JOB_NAME.indexOf("windows") != -1) {
+    SKIP_SIGNING_DEFAULT = false
+}
```

Comment:
Those 4 lines can be replaced with
```groovy
def SKIP_SIGNING_DEFAULT = ! JOB_NAME.contains("windows")
```

Reply:
Resolved in ace27b1ed20ddd91dc3c528712d6f37740113108.